src/d3d12/D3D12_Functions.cpp

```diff
@@ -497,13 +495,13 @@ void D3D12::PrepareUpdate()
 
     for (auto i = 0; i < drawData.CmdListsCount; ++i)
         IM_DELETE(drawData.CmdLists[i]);
-    delete[] drawData.CmdLists;
-    drawData.CmdLists = nullptr;
     drawData.Clear();
 
     drawData = *ImGui::GetDrawData();
 
-    auto** copiedDrawLists = new ImDrawList*[drawData.CmdListsCount];
+    ImVector<ImDrawList*> copiedDrawLists;
```

Comment:
this is also wrong, you pass in address on the stack! this is not memory safe at all, you again leak memory i  a very bad way

Reply:
this had changed in the ImGui structure we use apparently, not obvious from changes